sklearn/linear_model/tests/test_sparse_coordinate_descent.py

```diff
@@ -267,3 +267,35 @@ def test_same_output_sparse_dense_lasso_and_enet_cv():
         assert_almost_equal(clfs.intercept_, clfd.intercept_, 7)
         assert_array_almost_equal(clfs.mse_path_, clfd.mse_path_)
         assert_array_almost_equal(clfs.alphas_, clfd.alphas_)
+
+
+def test_same_multiple_output_sparse_dense():
+    for normalize in [True, False]:
+        l = ElasticNet(normalize=normalize)
+        X = [[0, 1, 2, 3, 4],
+             [0, 2, 5, 8, 11],
+             [9, 10, 11, 12, 13],
+             [10, 11, 12, 13, 14]]
+        y = [[1, 2, 3, 4, 5],
+             [1, 3, 6, 9, 12],
+             [10, 11, 12, 13, 14],
+             [11, 12, 13, 14, 15]]
+        l.fit(X, y)
+        ignore_warnings(l.fit)(X, y)
+        sample = np.array([1, 2, 3, 4, 5]).reshape(1, -1)
+        predict_dense = l.predict(sample)
+
+        l_sp = ElasticNet(normalize=normalize)
+        X_sp = sp.coo_matrix([[0, 1, 2, 3, 4],
+                              [0, 2, 5, 8, 11],
+                              [9, 10, 11, 12, 13],
+                              [10, 11, 12, 13, 14]])
+        y_sp = [[1, 2, 3, 4, 5],
+                [1, 3, 6, 9, 12],
+                [10, 11, 12, 13, 14],
+                [11, 12, 13, 14, 15]]
+        ignore_warnings(l_sp.fit)(X_sp, y_sp)
+
```

Comment:
nitpicky, but i'd remove the newline on line 298 and add one after you assign `predict_sparse` so the cases are symmetrical and the `assert` is it's own block